src/app/people/people.component.html

```diff
@@ -20,6 +20,7 @@ <h1>Star Wars <small>Encyclopedia</small></h1><hr/>
               <tr app-person *ngFor="let person of people | async" [person]="person"></tr>
             </tbody>
           </table>
+          <app-spinner *ngIf="showSpinner"></app-spinner>
```

Comment:
Consider implementing spinner for all page, that will disable user interaction. And in order to use it inject service:
```javascript
spinnerService.enable();
spinnerService.disable();
```